shared/src/main/scala/scala/xml/Utility.scala

```diff
@@ -105,7 +104,7 @@ object Utility extends AnyRef with parsing.TokenTests {
     val escMap = pairs map { case (s, c) => c -> ("&%s;" format s) }
     val unescMap = pairs ++ Map("apos" -> '\'')
   }
-  import Escapes.{ escMap, unescMap }
+  import Escapes.unescMap
```

Comment:
No, I'm wrong.  The implementation really didn't use `escMap`.   It's that #102 wanted to start using it again, and couldn't find it.  We're good here!